### PR TITLE
Localizes `content/bn/containers-as-a-service.md` to Bengali

### PR DESCRIPTION
**Pull request by PratikDey**
Signed-off-by: [Pratik Dey](pratikdey.pro@gmail.com)
Localizes `content/bn/containers-as-a-service.md` to Bengali
This is an initial PR as may need some improvements!
Tracking issue: https://github.com/cncf/glossary/issues/410

### CODE REVIEW

File: content/bn/containers-as-a-service.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+CaaS প্রদানকারীরা একটি ফ্রেমওয়ার্ক বা অর্কেস্ট্রেশন প্ল্যাটফর্ম অফার করে যা
+কী আইটি ফাংশনগুলিকে স্বয়ংক্রিয় করে, যার উপর কন্টেনার স্থাপন করা হয় এবং পরিচালিত করা হয়।
+কারণ ব্যবহারকারীরা তাদের প্রয়োজনীয় সংস্থানগুলিই কেনেন (শিডিউল করার ক্ষমতা, লোড ব্যালেন্সিং ইত্যাদি),
+তারা অর্থ সঞ্চয় এবং দক্ষতা বৃদ্ধি করে।
+কন্টেইনার দ্রুত বিকাশের জন্য সামঞ্জস্যপূর্ণ পরিবেশ তৈরি করে এবং
+[ক্লাউড-নেটিভ অ্যাপ্লিকেশানগুলি](/bn/cloud-native-apps/) সরবরাহ করে যা যেকোনো জায়গায় চলতে পারে।
+
+## এটা যেসব সমস্যাতে দৃষ্টিপাত করে
+
+CaaS ছাড়া, সফ্টওয়্যার ডেভেলপমেন্ট দলগুলিকে স্থাপন, পরিচালনা এবং নিরীক্ষণ করতে হবে
```
Comment (mitul3737) [inline content/bn/containers-as-a-service.md:23]: Kindly form a suitable sentence here.

File: content/bn/containers-as-a-service.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+tags: ["প্ল্যাটফর্ম", "", ""]
+---
+
+## এটা কি
+
+কন্টেইনার এজ এ সার্ভিস (CaaS) হল একটি ক্লাউড পরিষেবা যা অ্যাপগুলি পরিচালনা এবং স্থাপনে সহায়তা করে
+[ধারক](/bn/container/)-ভিত্তিক [বিমূর্ততা](/bn/abstraction) ব্যবহার করে।
+এই পরিষেবাটি প্রাঙ্গনে(on-premises) বা ক্লাউডে স্থাপন করা যেতে পারে।
+
+CaaS প্রদানকারীরা একটি ফ্রেমওয়ার্ক বা অর্কেস্ট্রেশন প্ল্যাটফর্ম অফার করে যা
```
Comment (mitul3737) [inline content/bn/containers-as-a-service.md:14]: Kindly form some meaning ful sentence from line 14-19

File: content/bn/containers-as-a-service.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+অন্তর্নিহিত অবকাঠামো যেখানে কন্টেইনার চলে।
+
+## এটা কিভাবে সাহায্য করে
+
+একটি CaaS প্ল্যাটফর্মে কন্টেইনারাইজড অ্যাপ্লিকেশন স্থাপন করার সময়,
+লগ এগ্রিগেশন এবং মনিটরিং টুলের মাধ্যমে ব্যবহারকারীরা সিস্টেম পারফরম্যান্সে দৃশ্যমানতা লাভ করে।
+CaaS-এ [অটো স্কেলিং](/bn/auto-scaling/) এবং অর্কেস্ট্রেশন পরিচালনার জন্য অন্তর্নির্মিত কার্যকারিতাও রয়েছে।
+এটি দলগুলিকে উচ্চ দৃশ্যমানতা এবং উচ্চ প্রাপ্যতা [ডিস্ট্রিবিউটেড সিস্টেম](/bn/distributed-systems/) তৈরি করতে সক্ষম করে।
+উপরন্তু, দ্রুত স্থাপনার অনুমতি দিয়ে, CaaS টিম ডেভেলপমেন্টের বেগ বাড়ায়।
+যেখানে কন্টেইনারগুলি একটি ধারাবাহিক স্থাপনার লক্ষ্য নিশ্চিত করে,
```
Comment (mitul3737) [inline content/bn/containers-as-a-service.md:33]: Kindly form a suitable sentence from line 33-35

**Review by mitul3737 — CHANGES_REQUESTED** [review]

**Review by Mouly22 — APPROVED** [review]

**Review by mitul3737 — APPROVED** [review]
lgtm

**Pull request merged**